SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+The following projects and versions are currently supported with security updates.
+
+| Package | Version | Supported          |
+| ------- | ------- | ------------------ |
+| `unstructured` | All versions   | :white_check_mark: |
+| `unstructured-api-tools` | All versions   | :white_check_mark: |
+| `pipeline-sec-filings` | All versions   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you find a security vulenerability in one of our projects, email us
```

Comment:
Should we emphasize that they _not_ open an issue?

Reply:
Added that in there